cmd/e2e-cleaner/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/e2e"
+	_ "k8s.io/klog/v2"
+)
+
+func init() {
+	flag.Usage = func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of %s:\n", os.Args[0])
+		flag.PrintDefaults()
+		fmt.Fprint(flag.CommandLine.Output(), "\n\n  Example Usage: go run ./cmd/e2e-cleaner/main.go -project my-project -run-id \"\"\n")
+	}
+}
+
+func main() {
+	if err := eMain(); err != nil {
```

Comment:
`eMain` doesn't seem to remove any errors. Please consider consolidating the two